### DISCUSSION TITLE
"stop" or "cancel" a giveaway

### DISCUSSION THREAD

**Discussion by Compositr** [Androz2091/discord-giveaways#416] [Ideas]
There should be a method to "stop" or "cancel" a giveaway, so that the giveaway ends without winners being drawn. Something like a permanent, irreversible pause.

Currently, I am using a workaround by pausing the giveaway forever:
```ts
giveaway
  .pause({
    content: ":x: This giveaway has been cancelled",
    unPauseAfter: Infinity
  }  as any /* Fix for incorrect typings in stable version */)
```

**Comment by imranbarbhuiya** [comment]
You can delete the giveaway

**Reply by Compositr** [reply]
Wdym? There is no delete method on the Giveaway class... do you mean deleting the message?

**Reply by imranbarbhuiya** [reply]
https://discord-giveaways.js.org/GiveawaysManager.html#delete
